tests/lean/runTacticMustCatchExceptions.lean.expected.out

```diff
@@ -1,27 +1,36 @@
-runTacticMustCatchExceptions.lean:2:25-2:28: error: tactic 'rfl' failed, equality expected
-  Nat.le 1 (a + b)
-a b : Nat
-⊢ 1 ≤ a + b
-runTacticMustCatchExceptions.lean:3:25-3:28: error: tactic 'rfl' failed, equality expected
-  Nat.le (a + b) b
-a b : Nat
-this : 1 ≤ a + b
-⊢ a + b ≤ b
-runTacticMustCatchExceptions.lean:4:25-4:28: error: tactic 'rfl' failed, equality expected
-  Nat.le b 2
-a b : Nat
-this✝ : 1 ≤ a + b
-this : a + b ≤ b
-⊢ b ≤ 2
-runTacticMustCatchExceptions.lean:9:18-9:21: error: tactic 'rfl' failed, equality expected
-  Nat.le 1 (a + b)
-a b : Nat
-⊢ 1 ≤ a + b
-runTacticMustCatchExceptions.lean:10:14-10:17: error: tactic 'rfl' failed, equality expected
-  Nat.le (a + b) b
-a b : Nat
-⊢ a + b ≤ b
-runTacticMustCatchExceptions.lean:11:14-11:17: error: tactic 'rfl' failed, equality expected
-  Nat.le b 2
-a b : Nat
-⊢ b ≤ 2
+runTacticMustCatchExceptions.lean:2:25-2:28: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  1 ≤ a + b : Prop
+runTacticMustCatchExceptions.lean:3:25-3:28: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  a + b ≤ b : Prop
+runTacticMustCatchExceptions.lean:4:25-4:28: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  b ≤ 2 : Prop
+runTacticMustCatchExceptions.lean:9:18-9:21: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  1 ≤ a + b : Prop
+runTacticMustCatchExceptions.lean:10:14-10:17: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  a + b ≤ b : Prop
+runTacticMustCatchExceptions.lean:11:14-11:17: error: type mismatch
+  Iff.rfl
+has type
+  ?m ↔ ?m : Prop
+but is expected to have type
+  b ≤ 2 : Prop
```

Comment:
This seems to be a slight regression of error reporting, but for something so elementary as `rfl` not irrelevant. 

If macro rules are used, which tactic decides about the error message? Last or first? Can we somehow register a `fail "trivial equality (or equivalence) expected"` or simply `fail "rfl failed"` or something like this as the last tactic to try with `rfl`?